ebpf/cmd/playground/main.go

```diff
@@ -35,6 +35,14 @@ import (
 
 var configFile = flag.String("config", "", "config file path")
 var server = flag.String("server", "http://localhost:4040", "")
+var discoverFreq = flag.Duration("discover.freq",
+	//5 * time.Second,
+	5*time.Hour, //todo remove
```

Comment:
TODO: Remove

Reply:
removed, thanks. I was benchmarking and wanted to exclude discovery